### PR TITLE
Fixed JetMET summary plot for HI DQM

### PR DESCRIPTION
**Pull request by jusaviin**
Backport #24993 to 10_3_X.

Connected histograms from HIJetValidation to the JetMET summary plot in case of pp_on_AA_2018 era. The JetMET summary plot was by default reading the pp histograms, which are disabled for HI. To give green light for jets in the summary plot, it is required that the mean pT is lower than 50 GeV. Also show "Not set" instead of "Bad" for blocks that are not connected for HI.

### CODE REVIEW

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
+1

**Comment by jusaviin** [comment]
backport of #24993

File: DQMOffline/JetMET/src/DataCertificationJetMET.cc [hunk 1]
```diff
@@ -608,38 +610,79 @@ DataCertificationJetMET::dqmEndJob(DQMStore::IBooker& ibook_, DQMStore::IGetter&
...
+
+
+  } else { // Histograms for protons
+
+    //Jet Phi histos
+    meJetPhi[0] = iget_.get(newHistoName+cleaningdir+jetAlgo+"PFJets/Phi_Barrel");
+    meJetPhi[1] = iget_.get(newHistoName+cleaningdir+jetAlgo+"PFJets/Phi_EndCap");
+    meJetPhi[2] = iget_.get(newHistoName+cleaningdir+jetAlgo+"PFJets/Phi_Forward");
+    meJetPhi[3] = iget_.get(newHistoName+cleaningdir+jetAlgo+"CaloJets/Phi");
+    //meJetPhi[4] = iget_.get(newHistoName+cleaningdir+"JetPlusTrackZSPCorJetAntiKt5/Phi");
```
Comment (vargasa) [inline DQMOffline/JetMET/src/DataCertificationJetMET.cc]: Is this removable? 
```suggestion
```

File: DQMOffline/JetMET/src/DataCertificationJetMET.cc [hunk 2]
```diff
@@ -608,38 +610,79 @@ DataCertificationJetMET::dqmEndJob(DQMStore::IBooker& ibook_, DQMStore::IGetter&
...
+
+    //Jet Eta histos
+    meJetEta[0] = iget_.get(newHistoName+cleaningdir+jetAlgo+"PFJets/Eta");
+    meJetEta[1] = iget_.get(newHistoName+cleaningdir+jetAlgo+"PFJets/Eta");
+    meJetEta[2] = iget_.get(newHistoName+cleaningdir+jetAlgo+"PFJets/EtaFirst");
+    meJetEta[3] = iget_.get(newHistoName+cleaningdir+jetAlgo+"CaloJets/Eta");
+    //meJetEta[4] = iget_.get(newHistoName+cleaningdir+"JetPlusTrackZSPCorJetAntiKt5/Eta");
```
Comment (vargasa) [inline DQMOffline/JetMET/src/DataCertificationJetMET.cc]: Is this removable?

Comment (jusaviin) [reply DQMOffline/JetMET/src/DataCertificationJetMET.cc]: I'd guess so. That is the pp part of the code that I did not touch in this commit (except by moving it inside the else block).

File: DQMOffline/JetMET/src/DataCertificationJetMET.cc (hunk 1 above)
Comment (jusaviin) [reply DQMOffline/JetMET/src/DataCertificationJetMET.cc]: I'd guess so. That is the pp part of the code that I did not touch in this commit (except by moving it inside the else block).

Comment (jusaviin) [reply DQMOffline/JetMET/src/DataCertificationJetMET.cc]: As this is a backport of #24993 and that PR is already merged to master, I would prefer not to remove these commented out parts of code here either for consistency.

File: DQMOffline/JetMET/src/DataCertificationJetMET.cc (hunk 2 above)
Comment (jusaviin) [reply DQMOffline/JetMET/src/DataCertificationJetMET.cc]: As this is a backport of #24993 and that PR is already merged to master, I would prefer not to remove these commented out parts of code here either for consistency.

File: DQMOffline/JetMET/src/DataCertificationJetMET.cc (hunk 1 above)
Comment (jusaviin) [reply DQMOffline/JetMET/src/DataCertificationJetMET.cc]: @vargasa Do you think it is better to keep the code consistent with #24993 or remove these two lines of commented code? It does no harm to remove them from the backport if necessary.

**Comment by jusaviin** [comment]
backport of #24993

File: DQMOffline/JetMET/src/DataCertificationJetMET.cc (hunk 1 above)
Comment (vargasa) [reply DQMOffline/JetMET/src/DataCertificationJetMET.cc]: @jusaviin Please double check nothing is broken in the repo history by removing them, and if so. I'd suggest to remove them. So we can keep the code cleaner :)

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**